com.varnamproject.Varnam.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The same as with desktop file.

Reply:
(+ icon)

Reply:
I've moved them now to upstream, thank you for the valuable inputs :)